### ISSUE TITLE
Core acceptance test fails

### ISSUE THREAD

**Issue by phil-davis** [owncloud/files_primary_s3#306]
Last night some core acceptance tests failed on files_primary_s3
These are failing also in encryption - see issue https://github.com/owncloud/encryption/issues/175

https://drone.owncloud.com/owncloud/files_primary_s3/1445/131/17
apiAll-32-31-latest-mariadb10.2-php7.1
and similar in https://drone.owncloud.com/owncloud/files_primary_s3/1445/99/17
apiAll-32-31-master-mariadb10.2-php7.1

```
--- Failed scenarios:

    /var/www/owncloud/testrunner/tests/acceptance/features/apiWebdavProperties/copyFile.feature:99
    /var/www/owncloud/testrunner/tests/acceptance/features/apiWebdavProperties/copyFile.feature:100

94 scenarios (92 passed, 2 failed)
679 steps (675 passed, 2 failed, 2 skipped)
```

```
  Scenario Outline: Copying file to a path with extension .part should not be possible                                              # /var/www/owncloud/testrunner/tests/acceptance/features/apiWebdavProperties/copyFile.feature:86
    Given using <dav_version> DAV path                                                                                              # FeatureContext::usingOldOrNewDavPath()
    When user "user0" copies file "/welcome.txt" to "/welcome.part" using the WebDAV API                                            # FeatureContext::userCopiesFileUsingTheAPI()
    Then the HTTP status code should be "400"                                                                                       # FeatureContext::thenTheHTTPStatusCodeShouldBe()
    And the DAV exception should be "OCA\DAV\Connector\Sabre\Exception\InvalidPath"                                                 # FeatureContext::theDavElementShouldBe()
    And the DAV message should be "Can`t upload files with extension .part because these extensions are reserved for internal use." # FeatureContext::theDavElementShouldBe()
    And the DAV reason should be "Can`t upload files with extension .part because these extensions are reserved for internal use."  # FeatureContext::theDavElementShouldBe()
    And user "user0" should see the following elements                                                                              # FeatureContext::userShouldSeeTheElements()
      | /welcome.txt |
    But user "user0" should not see the following elements                                                                          # FeatureContext::userShouldSeeTheElements()
      | /welcome.part |

    Examples:
      | dav_version |
      | old         |
        /remote.php/webdav/welcome.txt is not in propfind answer but should
      | new         |
        /remote.php/dav/files/user0/welcome.txt is not in propfind answer but should
```

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/files_primary_s3/1445/115/17
apiAll-32-15-latest-mariadb10.2-php7.1
```
--- Failed scenarios:

    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:250
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:251
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:266
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:267
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:280
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:281
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:297
    /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:298

76 scenarios (68 passed, 8 failed)
963 steps (923 passed, 8 failed, 32 skipped)
```

e.g.
```
  Scenario Outline: user shares a file with file name longer than 64 chars to another user                                                         # /var/www/owncloud/testrunner/tests/acceptance/features/apiShareManagementBasic/createShare.feature:240
    Given using OCS API version "<ocs_api_version>"                                                                                                # FeatureContext::usingOcsApiVersion()
    And user "user1" has been created with default attributes and without skeleton files                                                           # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
    And user "user0" has moved file "welcome.txt" to "aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog.txt"                      # FeatureContext::userHasMovedFile()
    When user "user0" shares file "aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog.txt" with user "user1" using the sharing API # FeatureContext::userSharesFileWithUserUsingTheSharingApi()
    Then the OCS status code should be "<ocs_status_code>"                                                                                         # OCSContext::theOCSStatusCodeShouldBe()
    And the HTTP status code should be "200"                                                                                                       # FeatureContext::thenTheHTTPStatusCodeShouldBe()
    And as "user1" file "aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog.txt" should exist                                      # FeatureContext::asFileOrFolderShouldExist()

    Examples:
      | ocs_api_version | ocs_status_code |
      | 1               | 100             |
        Failed asserting that 404 matches expected 201.
      | 2               | 200             |
        Failed asserting that 404 matches expected 201.
```

**Comment by phil-davis** [comment]
core PR https://github.com/owncloud/core/pull/36759 was merged yesterday. Other core PRs merged yesterday were test changes, but I do not immediately see what core change would have broken these tests.

I just merged files_primary_s3 PR #303 which bumps some dependencies in files_primary_s3. Probably the CI after merge will get the same fails - let's see.
